tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -27,9 +28,7 @@ class AppServicePlan(ArmResourceManager):
     :example:
 
     Find all App Service Plans that are of the Basic sku tier.
-
```

Comment:
New lines are required for `code-block` directive

Reply:
Thanks!